runtime/doc/diagnostic.txt

```diff
@@ -440,6 +440,9 @@ Lua module: vim.diagnostic                                    *diagnostic-api*
                       specified line number.
       • {severity}?   (`vim.diagnostic.SeverityFilter`) See
                       |diagnostic-severity|.
+      • {enabled}?    (`boolean`, default: `nil`) Limit diagnostics to only
+                      enabled or disabled. If nil, all diagnostics are
+                      returned.
```

Comment:
The way this is worded is kind of ambiguous: if `enabled` is `true` and `severity` is also set, is `severity` ignored?

Reply:
It wouldn't be ignored, but both filters would be applied: https://github.com/neovim/neovim/blob/af8f34b0ed9b113da0077a322a73bca8a6263ace/runtime/lua/vim/diagnostic.lua#L851-L852

Reply:
Let's clarify that in the docs then, because that's not immediately obvious from "all diagnostics are returned".

Also a link to `|vim.diagnostic.enable()|` would be good to have here so that users know what "enabled" and "disabled" mean.

Reply:
Updated to:
https://github.com/neovim/neovim/blob/20edcbbd9ca78de44e1e8176590a8303b4405620/runtime/lua/vim/diagnostic.lua#L1362-L1365